site/src/components/DeploySettingsLayout/DeploySettingsLayout.tsx

```diff
@@ -50,13 +45,13 @@ export const DeploySettingsLayout: FC = () => {
         <Stack className={styles.wrapper} direction="row" spacing={6}>
           <Sidebar />
           <main className={styles.content}>
-            {deploymentValues ? (
+            {deploymentConfigQuery.data && deploymentDAUsQuery.data ? (
               <DeploySettingsContext.Provider
                 value={{
-                  deploymentValues,
-                  getDeploymentValuesError,
-                  deploymentDAUs,
-                  getDeploymentDAUsError,
+                  deploymentValues: deploymentConfigQuery.data,
+                  getDeploymentValuesError: deploymentConfigQuery.error,
```

Comment:
I'm a little confused about the `get` prefixes here. I'd expect the names to correspond to getter methods/functions, but if the values are coming straight from one of useQuery's returned-out objects, wouldn't they be the direct `error` properties themselves?

Reply:
💯 this is a "legacy" thing on how we were using XState but I think makes sense to remove it since we now use a different verbiage.